### PR TITLE
[#41] Pass SSH parameters to daemon

### PR DESCRIPTION
**Pull request by mdelapenya**
Please modify what you consider. Thanks

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -31,10 +31,20 @@ var sessionKey = 'sid';
...
+    remoteUser: program.remoteUser,
+    remotePort: program.remotePort
+};
+
 if (program.daemonize) {
     daemonize(__filename, program, {
         doAuthorization: doAuthorization,
-        doSecure: doSecure
+        doSecure: doSecure,
+        sshOptions: sshOptions
```
Comment (mthenw) [inline index.js]: could yo do here something similar to `doSecure`? e.g. 
```
var doSSL = !!program.remoteHost;
```

File: lib/daemonize.js [hunk 2]
```diff
@@ -35,6 +35,14 @@ module.exports = function (script, params, options) {
         );
     }
 
+    if (options.sshOptions) {
```
Comment (mthenw) [inline lib/daemonize.js]: ``` if (options.doSSL) {```

**Comment by mdelapenya** [comment]
Hey @mthenw, I updated the pull. Hope it fits project needs :)

Cheers!

**Comment by mthenw** [comment]
@mdelapenya thank you very much! In an hour I will publish new version to npm.

**Comment by mdelapenya** [comment]
Very very nice! My first node contribution!!!

**Pull request merged**